src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -318,6 +318,12 @@ try
         rect.top = rect.bottom - 1;
         break;
     }
+    case CursorType::DoubleUnderscore:
+    {
+        rect.top = rect.bottom - 3;
+        paintType = CursorPaintType::HorizontalOnly;

```

Comment:
Frankly, I'd rather we _not_ introduce another `CursorPaintType` just for this one cursor style. I think it might be easier to just use `rect` to set up one of the lines to be drawn, and in this case, just manually call `d2dContext->DrawRectangle(..., brush.Get());` with another rect that you set up for the other line.